### ISSUE TITLE
Device will sometimes hang on a black screen when rebooting the userspace

### ISSUE THREAD

**Issue by gerbil-snot** [pwn20wndstuff/Undecimus#2033]
**Describe the bug**
*Tested with all tweaks disabled via iCleaner*
I tested this 10 times, 10% of the time the device will not show the apple logo when rebooting the userspace and instead hang on a black screen for a couple minutes, then reboot. 

**To Reproduce**
1. Try to jailbreak with tweaks enabled (all of my tweaks were disabled)
2. Test several times, 1/10 times for me the bug presented itself.

**Device (please complete the following information):**
 - iOS Version: 13.5
 - iOS Device: iPhone Xs Max 
 - unc0ver Version: 5.0.1

**Place an "x" between the brackets if true:**
 - [X] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [X] this issue is present after a rootfs restore
 - [X] this issue is present on the latest version of unc0ver

**Comment by SarKaa** [comment]
Yes this happened to me as well, but it didn't affect the jailbreak, it just hangs every now and then, leading to a reboot, but jailbreaking again fixes it

**Comment by Hamlock-maneuverr** [comment]
Pirated tweaks can cause this behavior. For example a cracked version of "Ares" will cause this.  uninstalling the cracked tweak should fix it.

**Comment by SarKaa** [comment]
The way I fixed it was by removing kalm
